db/migrate/20171010205831_add_created_by_id_to_reservations.rb

```diff
@@ -0,0 +1,6 @@
+class AddCreatedByIdToReservations < ActiveRecord::Migration
+  def change
+    add_column :reservations, :created_by_id, :integer
+    add_foreign_key :reservations, :users, column: :created_by_id
```

Comment:
I think we need to manually add the index because, well, oracle. Oracle doesn't automatically add an index for foreign keys (or at least it didn't at one point). Might be worth confirming before we merge this.

Reply:
From a quick google search, it sounds like Oracle still doesn't automatically create the index. Added one.